generate_data/netcdf/src/main/java/zarr_implementations/jzarr/App.java

```diff
@@ -0,0 +1,130 @@
+package zarr_implementations.jzarr;
+
+
+import com.bc.zarr.ArrayParams;
```

Comment:
Next step, @JacinthEsther, will be to replace these imports with the classes that come from netcdf-java rather than those that come from jzarr.